sdk/iOS/OoyalaSkinSDK/React/OOReactSkinModel.m

```diff
@@ -388,7 +400,20 @@ - (void)handleVolumeChanged:(float)volume {
 }
 //OS: called only by OOReactSkinBridgeModuleMain
 - (void)setEmbedCode:(NSString *)embedCode {
-  [self.player setEmbedCode:embedCode];
+  
+#warning: old API. Remove when SDK version > 4.46.0_GA. If you need to use old API, uncomment -playForJustChangedItem in -bridgeCurrentItemChangedNotification: of 'OOSkinPlayerObserver'
+  //[self.player setEmbedCode:embedCode];
+  
+  //new API from OOyalaSDK. Available > 4.46.0_GA
+  __weak OOReactSkinModel *weakSelf = self;
+  [self.player setEmbedCode:embedCode withCallback:^(OOOoyalaError *error) {
+    NSLog(@"✅ Got callback. Is setEmbedCode successfull: [%d]", (error == nil));
```

Comment:
Same about `NSLog` and `true/false/YES/NO`

Reply:
Done